benchmark/run_from_branch.sh

```diff
@@ -10,14 +10,19 @@ cd $TMP_DIR
 echo "Installing the dependencies."
 pip install pipenv
 # this is to install benchmark dependencies
+pipenv --rm
 pipenv install --dev --skip-lock
 # this is to install the AEA in the Pipenv virtual env
-pipenv run pip install --upgrade aea[all]=="0.10.1"
+pipenv run pip install --upgrade .[all] -y
```

Comment:
Installing all the development environment is overkill imo.

Reply:
it's better to have whole environment